server/routes/admin/html.py

```diff
@@ -25,8 +26,14 @@
 from flask import render_template
 from flask import request
 
+from shared.lib import constants
+
 bp = Blueprint('admin', __name__, url_prefix='/admin')
 
+_CUSTOM_EMBEDDING_INDEX = 'user_all_minilm_mem'
```

Comment:
Consider adding comments on what these are and why hardcoding is ok.

Reply:
done